pages/apim/3.x/user-guide/publisher/manage-api.adoc

```diff
@@ -294,3 +294,19 @@ You can also decide to highlight one API. This API will be located at the top of
 
 image::apim/3.x/api-publisher-guide/manage-apis/graviteeio-manage-apis-categories-4.png[]
 image::apim/3.x/api-publisher-guide/manage-apis/graviteeio-manage-apis-categories-4.png[]
+
+== Organize your APIs with labels
+You can use `Labels` to add tags to your APIs. The only purpose of labels is to sort APIs so publishers can easily find the APIs they need in the Management UI.
```

Comment:
rewrite 'the Management UI' as 'APIM Management'.

Reply:
I replaced Management UI by APIM since the M stands for Management. it seems to be redundant.
what do you think ?

Reply:
APIM is ok, but APIM Management is the component name (these were updated a week or so ago - you can see the new component names in the updated documentation on the Gravitee site).

Reply:
it is fixed :)